R/zzz.R

```diff
@@ -15,3 +15,12 @@ globalVariables(c(
 #' @importFrom magrittr %>%
 #' @usage lhs \%>\% rhs
 NULL
+
+.onLoad <- function(libname, pkgname) {
+  if (!requireNamespace("reticulate", quietly = TRUE)) {
+    warning("Initialize TF with 'reticulate::py_available(TRUE)'", call. = FALSE)
+    invisible(FALSE)
+  }
+
+  invisible(reticulate::py_available(TRUE))
+}
```

Comment:
This seemed to solve my issues with having to run `load_model()` twice... Not sure whether this was platform specific, but it doesn't seem to break anything elsewhere

Reply:
This should be fixed with dev reticulate